book/reference/IS2-resources.md

```diff
@@ -0,0 +1,16 @@
+# ICESat-2 Resources
+
+A large number of resources exist for helping you get and work with ICESat-2 data.
+These range from packages and tutorials created by NSIDC, the DAAC that manages ICESat-2 data,
+to community created libraries (such as icepyx and SlideRule) to individual GitHub repositories
```

Comment:
Minor suggestion: Want to link to icepyx and sliderule here too? A new reader might be curious on what those are.

Reply:
Thanks for the suggestion!